packages/widget-playground/src/store/widgetConfig/utils/getConfigOutput.ts

```diff
@@ -0,0 +1,19 @@
+import type { WidgetConfig, WidgetTheme } from '@lifi/widget';
+
+export const getConfigOutput = (
+  config: Partial<WidgetConfig>,
+): Partial<WidgetConfig> => {
+  const { playground, ...theme } = config.theme as WidgetTheme;
```

Comment:
filtering out the playground property here before output it to the code editor - but I just want to double check that this is purely used for the playground and will never be used in the widget itself?

Reply:
Yes, that property is for playground only. For now, it has only a background color for the widget, but maybe we will extend it with something else.